--- a/devLib/Makefile	2015-09-08 16:30:40.000000000 +0100
+++ b/devLib/Makefile	2015-09-08 16:58:18.569199557 +0100
@@ -37,7 +37,7 @@
 #DEBUG	= -g -O0
 DEBUG	= -O2
 CC	= gcc
-INCLUDE	= -I.
+INCLUDE	= -I. -I../wiringPi/
 DEFS	= -D_GNU_SOURCE
 CFLAGS	= $(DEBUG) $(DEFS) -Wformat=2 -Wall -Winline $(INCLUDE) -pipe -fPIC
 
@@ -73,6 +73,7 @@
 .c.o:
 	$Q echo [Compile] $<
 	$Q $(CC) -c $(CFLAGS) $< -o $@
+	@ln -sf libwiringPiDev.so.$(VERSION) libwiringPiDev.so
 
 .PHONY:	clean
 clean:
@@ -113,6 +114,15 @@
 	install -m 0755 libwiringPiDev.so.$(VERSION)				~/wiringPi/debian-template/wiringPi/usr/lib/libwiringPiDev.so.$(VERSION)
 	ln -sf ~/wiringPi/debian-template/wiringPi/usr/lib/libwiringPiDev.so.$(VERSION)	~/wiringPi/debian-template/wiringPi/usr/lib/libwiringPiDev.so
 
+.PHONY:	install-fedora
+install-fedora:	$(DYNAMIC)
+	@echo "[Install Headers]"
+	@install -m 0755 -d						$(DESTDIR)$(PREFIX)/include/wiringPi
+	@install -m 0644 $(HEADERS)					$(DESTDIR)$(PREFIX)/include/wiringPi
+	@echo "[Install Dynamic Lib]"
+	@install -m 0755 -d						$(DESTDIR)$(PREFIX)/lib
+	@install -m 0755 libwiringPiDev.so.$(VERSION)			$(DESTDIR)$(PREFIX)/lib/libwiringPiDev.so.$(VERSION)
+
 .PHONY:	uninstall
 uninstall:
 	$Q echo "[UnInstall]"
--- a/gpio/Makefile	2015-09-08 16:30:40.000000000 +0100
+++ b/gpio/Makefile	2015-09-08 16:46:06.687665762 +0100
@@ -80,6 +80,14 @@
 	$Q mkdir -p		$(DESTDIR)$(PREFIX)/man/man1
 	$Q cp gpio.1		$(DESTDIR)$(PREFIX)/man/man1
 
+.PHONY:        install-fedora
+install-fedora:        gpio
+	@echo "[Install]"
+	@install -m 0755 -d	$(DESTDIR)$(PREFIX)/bin
+	@install -m 0755 gpio	$(DESTDIR)$(PREFIX)/bin
+	@mkdir -p		$(DESTDIR)$(PREFIX)/share/man/man1
+	@install -m 0644 gpio.1	$(DESTDIR)$(PREFIX)/share/man/man1
+
 .PHONY:	install-deb
 install-deb:	gpio
 	$Q echo "[Install: deb]"
--- a/wiringPi/Makefile	2015-09-08 18:05:24.032753612 +0100
+++ b/wiringPi/Makefile	2015-09-08 18:05:46.012500667 +0100
@@ -83,6 +83,7 @@
 .c.o:
 	$Q echo [Compile] $<
 	$Q $(CC) -c $(CFLAGS) $< -o $@
+	@ln -sf libwiringPi.so.$(VERSION) libwiringPi.so
 
 
 .PHONY:	clean
@@ -137,6 +138,15 @@
 	install -m 0755 libwiringPi.so.$(VERSION)				~/wiringPi/debian-template/wiringPi/usr/lib/libwiringPi.so.$(VERSION)
 	ln -sf ~/wiringPi/debian-template/wiringPi/usr/lib/libwiringPi.so.$(VERSION)	~/wiringPi/debian-template/wiringPi/usr/lib/libwiringPi.so
 
+.PHONY:	install-fedora
+install-fedora:	$(DYNAMIC)
+	@echo "[Install Headers]"
+	@install -m 0755 -d						$(DESTDIR)$(PREFIX)/include/wiringPi
+	@install -m 0644 $(HEADERS)					$(DESTDIR)$(PREFIX)/include/wiringPi
+	@echo "[Install Dynamic Lib]"
+	@install -m 0755 -d						$(DESTDIR)$(PREFIX)/lib
+	@install -m 0755 libwiringPi.so.$(VERSION)			$(DESTDIR)$(PREFIX)/lib/libwiringPi.so.$(VERSION)
+
 .PHONY:	uninstall
 uninstall:
 	$Q echo "[UnInstall]"
